packages/components/src/TagList/TagList.tsx

```diff
@@ -37,6 +42,7 @@ const StyledArrowButton = styled(Tag)`
 export const TagList: React.FC<TagListProps> = ({
   initialCollapsed,
   collapseAfter,
+  small,
```

Comment:
Du könntest das small auch direkt hier initialisieren, dann muss der Check unten nicht mehr doppelt sein

```suggestion
  small = false,
```